alphadia/workflow/peptidecentric.py

```diff
@@ -514,6 +472,68 @@ def search_parameter_optimization(self):
 
         self.save_managers()
 
+    def process_batch(self):
```

Comment:
could all of these new methods be private? also, please add type hints and (concise) docstrings ..